addons/main/3den/feature_persistence.hpp

```diff
@@ -1,4 +1,4 @@
-﻿class RR_commons_persistence {
+class RR_commons_persistence {
```

Comment:
What changed?

Reply:
Diff prob. due to prev. falty file encoding